buildconfigs/20-ovn-kubernetes-base.yaml

```diff
@@ -14,7 +14,8 @@ spec:
     type: Docker
     dockerStrategy:
       imageOptimizationPolicy: SkipLayers
-      dockerfilePath: Dockerfile.base.rhel9
+      # Dockerfile.rhel9 is suitable for 4.13 only
```

Comment:
4.13 is the current stable and we should start rolling out changes to onboard 4.14 builds: this change leads again to a question we were discussing some time ago.

What versions should we support in this repo? 

@vrutkovs should we branch and create a release-4.12, release-4.13 and keep the master branch for 4.14?

Reply:
Yeah, lets create release-4.13 branch at least. Not sure if we have capacity to actively support 4.12 builds tbh, but worth having for historical purposes at least

Reply:
branching is suitable here